packages/dialtone/lib/build/less/components/skeleton.less

```diff
@@ -69,3 +71,27 @@
     width: 100%;
   }
 }
+
+//  ============================================================================
+//  @   SKELETON PARAGRAPH
+//  ----------------------------------------------------------------------------
+.d-skeleton-paragraph {
+  width: 100%;
+  margin-bottom: var(--dt-space-450);
+
+  &:last-child {
+    margin-bottom: var(--dt-space-0);
+  }
+}
+
+//  ============================================================================
+//  @   SKELETON TEXT
+//  ----------------------------------------------------------------------------
+.d-skeleton-text {
+  height: 80%;
+  border-radius: var(--dt-size-radius-200);
```

Comment:
`d-h8, d-bar2`

Reply:
hmmm
`height: 80%;` should be `height: var(--dt-size-400)`, what Im missing?
border-radius its ok

Reply:
You are right! Bad copy pasting 🤪